Sources/Defaults/Defaults.swift

```diff
@@ -65,6 +65,18 @@ public final class Defaults {
 			suite.removeObject(forKey: key)
 		}
 	}
+	
+	public func clear<T: Codable>(key: Defaults.Key<T>, suite: UserDefaults = .standard) {
+		for key in suite.dictionaryRepresentation().keys.filter({ $0 == key.name }) {
```

Comment:
Use the “where” keyword instead of “filter”.

Reply:
I have to apologize but I am not 100% sure what you mean. Were you thinking of `first(where: ...`?

Reply:
```swift
for suiteKey in suite.dictionaryRepresentation().keys where suiteKey == key {
```